### PR TITLE
Set up AIPresenter class

### PR DESCRIPTION
**Pull request by ephraimacquah**

### CODE REVIEW

File: ai_presenter/ai_presenter.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
+from ai_presenter.database import Database
+import logging
+
+
+class TextAi:
```
Comment (lpabon) [inline ai_presenter/ai_presenter.py:5]: Move all classes that are not AiPresenter to another file

File: ai_presenter/ai_presenter.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+class TextFake(TextAi):
+    def __init__(self, db: Database):
+        super().__init__(db)
+
+    def generate(self):
+        pass
+
+    def send(self, text) -> str:
+        logging.info(f"textfake: Sending {text}")
```
Comment (ephraimacquah) [inline ai_presenter/ai_presenter.py:26]: change "textfake" to match class name "TextFake"

File: something.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+John Doe, a charismatic detective and Jane Smith, a brilliant scientist are at a crime scene in a cabin. Build a scene about them with this dialogue: actor: John Doe says, "I need to examine the evidence carefully." Jane Smith says, "Let's analyze the DNA samples." Be sure to expand on this dialogue and include many details
```
Comment (lpabon) [inline something.txt:1]: Delete this file

**Review by lpabon — CHANGES_REQUESTED** [review]

File: ai_presenter/ai_presenter.py (hunk 2 above)
Comment (lpabon) [reply ai_presenter/ai_presenter.py:26]: A possible addition in a future release is for each class to have a `self.log = logging...` that way any of the methods can use `self.log(...)` and have it set up with the prefix

File: ai_presenter/database.py [hunk 4]
```diff
@@ -12,17 +13,34 @@ def __init__(self, data):
...
 class Location:
     def __init__(self, data):
         self.name = data['name']
         self.type = data['type']
-        self.lighting = data['lighting']
\ No newline at end of file
+        self.lighting = data['lighting']
+
+
+class Config:
```
Comment (lpabon) [inline ai_presenter/database.py:34]: Move Config classes to `config/` directory and have `config.py`, `ai.py`, and `chatgpt.py`

**Review by lpabon — APPROVED** [review]
We can do the final changes after.

**Pull request merged**